### PR TITLE
tarsnapaccount.cpp: include QHeaderView for Qt 5.11.0 compatibility

### PR DESCRIPTION
**Pull request by apjanke**
Fixes https://github.com/Tarsnap/tarsnap-gui/issues/179

### CODE REVIEW

File: src/tarsnapaccount.cpp [hunk 1]
```diff
@@ -7,6 +7,7 @@
 #include <QNetworkRequest>
 #include <QSettings>
 #include <QTableWidget>
+#include <QHeaderView>
```
Comment (shinnok) [inline src/tarsnapaccount.cpp]: Can you move the new line atop of QMessageBox, to maintain alphabetical sort order for header includes? :)

**Comment by gperciva** [comment]
Thanks, @apjanke!

**Comment by apjanke** [comment]
Excellent. You're welcome.

What's your release policy? Any chance of a 1.0.1 release to pick up this and any other recent fixes? Or do we need to wait for the next regular minor release?

**Comment by gperciva** [comment]
I was hoping to get some more automated tests in place before looking at making a release, but I agree that it would be good to get these build fixes out sooner rather than later.  I'll aim to have a release candidate in a few days.

**Comment by apjanke** [comment]
Sweet. :+1:

**Comment by ilovezfs** [comment]
What is the status of https://github.com/Tarsnap/tarsnap-gui/releases/tag/v1.0.1 ? It doesn't seem to be designated as an actual release: https://github.com/Tarsnap/tarsnap-gui/releases/latest

**Comment by gperciva** [comment]
Hi @ilovezfs.

The release is almost ready, but I forgot some stuff:
https://github.com/Tarsnap/tarsnap-gui/pull/182

I didn't realize that pushing a tag called `v1.0.1` would make github automatically add it to the release list.  :(

**Comment by gperciva** [comment]
Hi @ilovezfs,

I've now made 1.0.1 official.  Please note that the tarball has changed; the `.tar.gz` sha256 is now `502e53a4eacb2795bfbe62b9e6ef236cce6fdb43cb15153c0fc4ad61c161eb97`.  This value has been updated in `util/tarsnap-gui.rb` in git master.

Release email:
http://mail.tarsnap.com/tarsnap-users/msg01456.html

Release on github:
https://github.com/Tarsnap/tarsnap-gui/releases/tag/v1.0.1

Sorry for the confusion over this release; it was a bit of a learning process.  :)

**Pull request merged**